src/app/zap-templates/zcl/data-model/chip/scene.xml

```diff
@@ -16,17 +16,17 @@ limitations under the License.
 -->
 <configurator>
   <bitmap name="NameSupportBitmap" type="bitmap8">
-      <cluster code="0x0005"/>
+      <cluster code="0x0062"/>
```

Comment:
Please wait for https://www.righto.com/?m=1 to merge before resolving this comment and merging

Reply:
@tcarmelveilleux wrong copy paste i think?

Reply:
Agreed, this will not be merged until the spec gets updated

Reply:
The proper link for the spec issue to resolve before this is [here](https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/8667)

Reply:
The Spec PR is merged, we can move forward with this one